css-editor/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "css-editor",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "gh-pages": "^2.2.0",
+    "prismjs": "^1.20.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-scripts": "3.4.1",
+    "styled-components": "^5.0.1"
+  },
+  "homepage": "http://superhackerboi.github.io/2020-04-04-Live-CSS-Editor",
```

Comment:
This link seem to be broken. Either you did not deploy with `npm run deploy` or the deploy is broken.